default.js

```diff
@@ -0,0 +1,6 @@
+const inliner = require('./index');
+
+module.exports = {
+  'svg': inliner('./', {}),
+  'inline-svg': inliner('./', {})
```

Comment:
I prefer the second, but I understand the appeal of the first and it's the example in the docs so I figured I'd just have both.